src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    public static final int TYPE = 0;
```

Comment:
Avoid using public static final fields for indexes, as this can lead to errors if the file format changes. Consider creating a separate DTO class to represent the parsed data and use an object mapper to convert lines into DTOs.